website/components/card-list/style.module.css

```diff
@@ -0,0 +1,57 @@
+.cardsWrapper {
+  display: grid;
+  column-gap: 40px;
+  row-gap: 40px;
+  grid-template-columns: repeat(auto-fill, minmax(272px, 1fr));
+
+  & .card {
+    border: 1px solid #dcdde0;
```

Comment:
Do we have a --gray variable for this value here?

Reply:
I checked but didn't see one in [the global colors](https://github.com/hashicorp/mktg-global-styles/blob/main/custom-properties/color.css)

Reply:
Often what I'll do here is just grab the most similar gray color; usually there's something visually close.

If not though, using a custom color is OK.

Reply:
👍 gray-5 looks very similar, we can go with that 
<img width="969" alt="Screen Shot 2021-09-28 at 2 52 44 PM" src="https://user-images.githubusercontent.com/20215887/135148431-d6cd2f5c-667a-4dd2-8f4f-8c4e818fd4ef.png">